starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/Shopping_List.java

```diff
@@ -0,0 +1,61 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+/**
+ * Created by ds on 10/25/16.
+ */
+
+public class Shopping_List {
+    private int mID;
+    private String mItem_name;
+    private String mDescription;
+    private double mPrice;
+    private String mFood_type;
+
+    public Shopping_List(int id, String item_name, String description, double price, String food_type) {
+        mID = id;
+        mItem_name = item_name;
+        mDescription = description;
+        mPrice = price;
+        mFood_type = food_type;
+    }
+
+    public int getId() {
+        return mID;
+    }
+
+    public void setId(int id) {
+        mID = id;
+    }
+
+    public String getItem_name() {
+        return mItem_name;
+    }
+
+    public void setItem_name(String item_name) {
+        mItem_name = item_name;
```

Comment:
Totally fine to have setters here, but I just want to make sure you know you don't always need to have setters. Sometimes you don't want an object's values to change after it's created, so you might prefer not to provide any setters.